### PR TITLE
feat: Old external CI deprecated warning

### PR DESCRIPTION
**Pull request by vivek-devtron**
# Description

Show deprecated warning on old external CI
Screenshot 2022-11-28 at 2 13 53 PM
Screenshot 2022-11-28 at 2 14 08 PM

Fixes # [AB#1284](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/1284)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
Manually tested by verifying with older CI

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/workflowEditor/DeprecatedWarningModal.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { ConfirmationDialog } from '../common'
+import warningIconSrc from '../../assets/icons/ic-warning-y6.svg'
+import { DEPRECATED_EXTERNAL_CI_MESSAGE, DOCUMENTATION } from '../../config'
+
+export default function DeprecatedWarningModal({ closePopup }: { closePopup: () => void }) {
+    const close = (): void => {
```
Comment (shivani170) [inline src/components/workflowEditor/DeprecatedWarningModal.tsx:7]: Could we please rename it to more specific

**Review by shivani170 — APPROVED** [review]

File: src/components/workflowEditor/DeprecatedWarningModal.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                <div className="fs-14 cn-7 mt-20 w-100">
+                    {DEPRECATED_EXTERNAL_CI_MESSAGE.LINE_THREE}&nbsp;
+                    <a className="dc__link" href={DOCUMENTATION.WEBHOOK_CI} rel="noreferrer noopener" target="_blank">
+                        {DEPRECATED_EXTERNAL_CI_MESSAGE.DOC_LINK_TEXT}
+                    </a>
+                </div>
+            </ConfirmationDialog.Body>
+            <ConfirmationDialog.ButtonGroup>
+                <div className="flex right">
+                    <button type="button" className="cta cancel" onClick={close}>
```
Comment (sohel-rp) [inline src/components/workflowEditor/DeprecatedWarningModal.tsx:25]: Can we use `closePopup` directly?

File: src/components/workflowEditor/DeprecatedWarningModal.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { ConfirmationDialog } from '../common'
+import warningIconSrc from '../../assets/icons/ic-warning-y6.svg'
+import { DEPRECATED_EXTERNAL_CI_MESSAGE, DOCUMENTATION } from '../../config'
+
+export default function DeprecatedWarningModal({ closePopup }: { closePopup: () => void }) {
```
Comment (sohel-rp) [inline src/components/workflowEditor/DeprecatedWarningModal.tsx:6]: move this to type

File: src/components/workflowEditor/DeprecatedWarningModal.tsx (hunk 2 above)
Comment (vivek-devtron) [reply src/components/workflowEditor/DeprecatedWarningModal.tsx:25]: No. The closePopup method expects some optional params.  If we use it directly then default param events will be included automatically.

Comment (sohel-rp) [reply src/components/workflowEditor/DeprecatedWarningModal.tsx:26]: ok

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**